2-multitenant/modules/env_baseline/README.md

```diff
@@ -17,13 +17,14 @@ The following resources are created:
 |------|-------------|------|---------|:--------:|
 | apps | Applications | <pre>map(object({<br>    ip_address_names = optional(list(string))<br>    certificates     = optional(map(list(string)))<br>  }))</pre> | n/a | yes |
 | billing\_account | The billing account id associated with the project, e.g. XXXXXX-YYYYYY-ZZZZZZ | `string` | n/a | yes |
+| cluster\_release\_channel | The release channel for the clusters | `string` | `"REGULAR"` | no |
 | cluster\_subnetworks | The subnetwork self\_links for clusters | `list(string)` | n/a | yes |
+| cluster\_type | Cluster types: STANDARD, STANDARD-NAP, AUTOPILOT | `string` | `"STANDARD-NAP"` | no |
```

Comment:
STANDARD and AUTOPILOT are documented publicaly, is STANDARD-NAP documented somewhere? it may need some more description somewhere to make it explicit what it means

Reply:
good point - added "(Standard with node auto-provisioning)"